lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -0,0 +1,235 @@
+module Pod
```

Comment:
This is where the project generation logic now lives. The only changes from the code that lived in `Installer` is that I broke out `add_system_framework_dependencies` and `create_project`, call order is maintained.